.travis.yml

```diff
@@ -26,7 +25,7 @@ addons:
 
 script:
   - eclint check * */* */cunit/*
-  - ./tools/build-install.sh --with-lazperf=/usr/local --with-libght=/usr/local && make check
-  - ./tools/build-install.sh --with-lazperf=/usr/local --without-libght && ./tools/valgrind.sh
+  - ./tools/build-install.sh --with-lazperf=/usr/local && make check
+  - ./tools/build-install.sh --with-lazperf=/usr/local && ./tools/valgrind.sh
```

Comment:
Now that we don't build with libght we no longer need to run build-install.sh twice.

Reply:
Fixed